### ISSUE TITLE
Notification not shown if many lines of text are grabbed

### ISSUE THREAD

**Issue by mauriva** [TheJoeFin/Text-Grab#453]
I use Text Grab (v. 4.4.1) on Windows 11 23H2 in Full screen mode and the "Show notification" option turned on.
Everything works as expected if a few lines of text are selected, but if the selection affects many more lines the notification window is not shown (but the text is still copied to the clipboard).
I did not understand whether this is an intended behavior or a bug.

**Comment by TheJoeFin** [comment]
That is not the intended behavior, and in my testing the notifications still work when I OCR a lot of text. You'll have to share more details and ideally a sample which I can use to test.

Are you using Text Grab installed from the Store, GitHub (self-contained or just the EXE)? Also what language are you using, and are you using Tesseract?

**Comment by mauriva** [comment]
Thanks for your answer.

Here are a few more details.
- PC: Surface 6 Pro, Intel i5-8250U, RAM 8Gb
- OS: Windows 11 Pro 23H2
- Text Grab v. 4.4.1 Self-contained
- Tesseract w64 5.3.3.20231005
- Language: English, Italian for Tesseract downloaded from https://github.com/tesseract-ocr/tessdata_best

I attach two images taken from the "About Text Grab" window.
If I select the area corresponding to the first image, the notification window is shown as expected.
If I select the area corresponding to the second image, the text is copied to the clipboard, but the notification window is not shown.
I have tried several times selecting the same areas and the result is every time the same.
In all the tests I used English with Tesseract, but the thing also happens in other contexts with Italian language (with Tesseract).

Image 1: works ok (notification shown)
![TGrab1](https://github.com/TheJoeFin/Text-Grab/assets/39453887/f2a8e395-dec2-4af3-af40-3250c5813e02)
Image 2: not works (notification not shown)
![TGrab2](https://github.com/TheJoeFin/Text-Grab/assets/39453887/dfde7949-f4ff-45a1-a6ed-21493d657f99)

**Comment by TheJoeFin** [comment]
When I use English or Italian Tesseract or Windows OCR I get a toast every time. Can you check the Windows Event Viewer and see if there is anything in there about failing to send a notification?

**Comment by mauriva** [comment]
> When I use English or Italian Tesseract or Windows OCR I get a toast every time. Can you check the Windows Event Viewer and see if there is anything in there about failing to send a notification?

I'm not sure what to look for in the Windows Event Viewer. Any suggestion on how to locate the error, if any?

**Comment by TheJoeFin** [comment]
The first place I'd check is under Windows Logs, then Application. See if there are any events related to the notification not sending.
![image](https://github.com/TheJoeFin/Text-Grab/assets/7809853/216b5c85-5548-4a68-b463-034d3309f07f)

**Comment by mauriva** [comment]
> The first place I'd check is under Windows Logs, then Application. See if there are any events related to the notification not sending.

I checked and it seems to me that there is no error message in the Windows Event Viewer.
Upon further testing, I noticed that if I select an area with a lot of text to grab, the notification is created, but the notification window does not appear.
I noticed this because the bell in the lower right corner, after processing, lights up and clicking on it to open the Windows 11 notification area also shows the last one generated by TextGrab.

If there are no other notifications linked to TextGrab, the window appears even if the area to be processed contains a lot of text, but if I have already performed some operations with TextGrab, the notification is created but not shown.

I don't know if this helps in any way.
